citrination_client/base/response_handling.py

```diff
@@ -0,0 +1,32 @@
+from errors import *
+
+def raise_on_response(response):
```

Comment:
Why make this a module function? It might be nice to make this Client-specific (default behavior in BaseClient, but overridable).

Reply:
Make a method on base client invoke this.

Reply:
👍